urllib3/contrib/pyopenssl.py

```diff
@@ -204,10 +207,12 @@ def _send_until_done(self, data):
                 continue
 
     def sendall(self, data):
+        if sys.version_info >= (2, 7) and not isinstance(data, memoryview):
```

Comment:
Would be better to check for the presence of `memoryview` rather than check `sys.version_info`.

Maybe something like this near the top?

```python
try:
    _ = memoryview
    has_memoryview = True
except NameError:
    has_memoryview = False